projects/goharbor/harbor/docker/linux/harbor-core/Dockerfile

```diff
@@ -0,0 +1,31 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-base
+FROM $BASE_IMAGE
+
+ARG TARGETARCH
+ARG TARGETOS
+
+RUN yum install -y tzdata shadow-utils >> /dev/null \
+    && yum clean all \
+    && groupadd -f -r -g 10000 harbor && useradd --no-log-init -r -m -g 10000 -u 10000 harbor \
+    && mkdir /harbor/
+
+HEALTHCHECK CMD curl --fail -s http://localhost:8080/api/v2.0/ping || curl -k --fail -s https://localhost:8443/api/v2.0/ping || exit 1
+COPY _output/make/photon/common/install_cert.sh /harbor/
+COPY _output/make/photon/core/entrypoint.sh /harbor/
+COPY ../harbor/src/core/views /harbor/views
```

Comment:
We dont usually do this. Could we copy these files into output like youve done for others.